governance/xc_admin/packages/xc_admin_common/src/cluster.ts

```diff
@@ -27,3 +27,22 @@ export function getMultisigCluster(cluster: PythCluster): Cluster | "localnet" {
       return cluster;
   }
 }
+
+export function getMaximumNumberOfPublishers(cluster: PythCluster) {
+  switch (cluster) {
+    case "mainnet-beta":
+      return 32;
+    case "devnet":
+      return 32;
+    case "testnet":
+      return 32;
+    case "pythnet":
+      return 32;
```

Comment:
Note: I think this is good, we should prevent a pythnet issue if there's a regression